README.md

```diff
@@ -80,71 +76,68 @@ When multiple proxy configurations exists, precedence is taken in this form:
 contentful config add --raw-proxy
 ```
 
-## :books: Documentation
-
-More detailed documentation for every command can be found in the [docs section](https://github.com/contentful/contentful-cli/tree/master/docs).
-
 ## :hammer_and_wrench: Development
 
-After installing the dependencies, there is a trick to get your version of the CLI tool available globally on your system:
-
-```sh
-npm link
-```
-
-This may collide with your already globally installed Contentful CLI. Make sure to remove that one first.
+1. Install dependencies
+   ```sh
+   npm i
+   ```
+2. To avoid development version colliding with your already installed Contentful CLI, change the command name in `package.json`
+   ```json
+     "bin": {
+       // "contentful": "bin/contentful.js"
```

Comment:
I think that since the highlight is set as json, and as `//` is invalid in json, it will be rendered with a red background.
But not a blocker, just that it may look weird on the rendered readme tho

Reply:
TIL: **```diff** does this:

```diff
 "bin": {
-     "contentful": "bin/contentful.js"
+     "ctfl": "bin/contentful.js"
 }
```